internals/osutil/mkdir_test.go

```diff
@@ -112,8 +112,82 @@ func (mkdirSuite) TestMakeParentsAndExistNotOK(c *check.C) {
 	c.Assert(err, check.ErrorMatches, `.*: file exists`)
 }
 
+func (mkdirSuite) TestChmod(c *check.C) {
+	oldmask := syscall.Umask(0022)
```

Comment:
Should this be `0o022` (and below)?

Reply:
Both works, I've seen both 0o000 and 0000. To be consistent with other `syscall.Umask` usages in Pebble (for example [here](https://github.com/canonical/pebble/blob/master/internals/daemon/daemon.go#L825)), I will keep it as it is.